main.go

```diff
@@ -82,10 +82,10 @@ func collectMetricsInBackground(config internal.ConfigStruct, logger internal.IL
 	}()
 }
 
-func FetchResourceIdToNameMapping(client *internal.OtcWrapper, namespaces []string) (map[string]string, error) {
+func FetchResourceIdToNameMapping(client *internal.OtcWrapper, config internal.ConfigStruct) (map[string]string, error) {
```

Comment:
I feel like we should maybe adapt the otc wrapper because this behaviour could be required for other namespaces too.

Reply:
As in fold the configstruct into the wrapper?